### PR TITLE
feat(editChat): improve messages edit

### PR DESCRIPTION
**Pull request by nickjjang**
AP-292

### CODE REVIEW

File: components/views/chat/message/edit/Edit.vue [hunk 1]
```diff
@@ -125,6 +153,26 @@ export default Vue.extend({
             messageBox.innerHTML = ''
           }
           break
+        case 'a':
```
Comment (Jekrimo) [inline components/views/chat/message/edit/Edit.vue:156]: Will you make Enums for all of these strings, Some may exist already too

File: components/views/chat/message/edit/Edit.vue [hunk 2]
```diff
@@ -125,6 +153,26 @@ export default Vue.extend({
...
+        case 'ArrowLeft':
+        case 'Up':
+        case 'ArrowUp':
+        case 'Right':
+        case 'ArrowRight':
+        case 'Down':
+        case 'ArrowDown':
+          return
+        case 'Control':
+        case 'Shift':
```
Comment (Jekrimo) [inline components/views/chat/message/edit/Edit.vue:174]: Will you make Enums for all of these strings, Some may exist already too

**Review by Jekrimo — CHANGES_REQUESTED** [review]
Will you make Enums for all of these strings, Some may exist already too

File: components/views/chat/message/edit/Edit.vue (hunk 2 above)
Comment (nickjjang) [reply components/views/chat/message/edit/Edit.vue:174]: Hi @Jekrimo , thanks for letting me know.
I'll put all variable into keybindings.ts

Comment (Jekrimo) [reply components/views/chat/message/edit/Edit.vue:174]: Perfect, thank you!

Comment (nickjjang) [reply components/views/chat/message/edit/Edit.vue]: Hi @Jekrimo I have fixed keybinding enums.

**Review by WanderingHogan — APPROVED** [review]

**Pull request merged**